Two ogr2ogr folders now

gecko2019 opened the pull request:
updated ogr2ogr scripts into two new folders, each has a readme.md

wburt commented:
The readme still has a bunch of linkages to QGIS docs. Clean that up before this request is merged.

gecko2019 commented:
OK, fixed stuff:
- Created  docs\getting_started_with_ogr
- Split gis-pantry\recipes\ogr2ogr into two folders, 'ogr2ogr_Python_only' and 'ogrFromBCGW_csv'
